### PR TITLE
Fix some style issues

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
No code change

### CODE REVIEW

File: crypto/evp/e_chacha20_poly1305.c [hunk 1]
```diff
@@ -346,7 +346,7 @@ static int chacha20_poly1305_ctrl(EVP_CIPHER_CTX *ctx, int type, int arg,
     case EVP_CTRL_COPY:
         if (actx) {
             if ((((EVP_CIPHER_CTX *)ptr)->cipher_data =
```
Comment (richsalz) [inline crypto/evp/e_chacha20_poly1305.c]: It would be better to break this up into two, the assignment and the if test.

Comment (FdaSilvaYY) [inline crypto/evp/e_chacha20_poly1305.c]: I agree; I tried, but I don't like the result,...
You need to introduce a new variable to store the cast result, or copy-paste it ...

Comment (richsalz) [inline crypto/evp/e_chacha20_poly1305.c]: Add a variable.  What’s there now is gross.  Casting the LHS is uncool ☺

**Comment by richsalz** [comment]
+1

File: apps/speed.c [hunk 2]
```diff
@@ -1073,25 +1072,26 @@ static int run_benchmark(int async_jobs, int (*loop_function)(void *), loopargs_
 
 
     for (i = 0; i < async_jobs && !error; i++) {
-        switch (ASYNC_start_job(&(loopargs[i].inprogress_job), loopargs[i].wait_ctx,
+        async = ASYNC_start_job(&(loopargs[i].inprogress_job), loopargs[i].wait_ctx,
```
Comment (levitte) [inline apps/speed.c]: I don't quite understand how assigning to `async` is a style fix.

File: apps/speed.c [hunk 3]
```diff
@@ -1169,28 +1169,29 @@ static int run_benchmark(int async_jobs, int (*loop_function)(void *), loopargs_
                 continue;
 #endif
 
-            switch (ASYNC_start_job(&(loopargs[i].inprogress_job), loopargs[i].wait_ctx,
+            async = ASYNC_start_job(&(loopargs[i].inprogress_job), loopargs[i].wait_ctx,
```
Comment (levitte) [inline apps/speed.c]: Same here

File: apps/speed.c (hunk 2 above)
Comment (FdaSilvaYY) [inline apps/speed.c]: It's a 80 col overflow, that I try to fix... ;)

Comment (levitte) [inline apps/speed.c]: ... with a line that's exactly the same length?

Comment (richsalz) [inline apps/speed.c]: but it's an assignment, not in the middle of an expression.  i think it's more clear.

Comment (levitte) [inline apps/speed.c]: Really?  Ok then

**Comment by levitte** [comment]
It seems, though, that this branch could do with a rebase on top of a fresher master

**Comment by levitte** [comment]
+1

**Comment by richsalz** [comment]
+1 re-confirmed.  go for it, @levitte :)

**Comment by levitte** [comment]
Done

**Pull request closed without merging**